### ISSUE TITLE
Images are broken in the discovery book

### ISSUE THREAD

**Issue by japaric** [rust-embedded/bookshelf#4]
At least in the bookshelf version. See: 

https://rust-embedded.github.io/bookshelf/discovery/04-meet-your-hardware/index.html

The standalone version of the discovery book does show the images:

https://rust-embedded.github.io/discovery/04-meet-your-hardware/README.html

**Comment by therealprof** [comment]
cf. https://github.com/rust-embedded/book/pull/38

**Comment by adamgreen** [comment]
I was playing around with linkchecker tonight to see if I could see why it doesn't detect these missing image URLs. It seems like it treats the absolute links as external links so doesn't bother to check them. I was able to get it to detect the bad links by using this command line:

```linkchecker --ignore-url "http.*" --check-extern doc```

I used the ```--ignore-url``` option too since linkchecker gets a bunch of internal errors if it actually tries to validate links to external websites when I ran it on my Ubuntu VM.
